packages/hmr/README.md

```diff
@@ -0,0 +1,136 @@
+<div align="center">
+
+![Sapphire Logo](https://cdn.skyra.pw/gh-assets/sapphire-banner.png)
+
+# @sapphire/plugin-hmr
+
+**Plugin for <a href="https://github.com/sapphiredev/framework">@sapphire/framework</a> for super-speed HMR.**
+
+[![GitHub](https://img.shields.io/github/license/sapphiredev/plugins)](https://github.com/sapphiredev/plugins/blob/main/LICENSE.md)
+[![codecov](https://codecov.io/gh/sapphiredev/plugins/branch/main/graph/badge.svg?token=QWL8FB16BR)](https://codecov.io/gh/sapphiredev/plugins)
+[![npm bundle size](https://img.shields.io/bundlephobia/min/@sapphire/plugin-hmr?logo=webpack&style=flat-square)](https://bundlephobia.com/result?p=@sapphire/plugin-hmr)
+[![npm](https://img.shields.io/npm/v/@sapphire/plugin-hmr?color=crimson&logo=npm&style=flat-square)](https://www.npmjs.com/package/@sapphire/plugin-hmr)
+
+</div>
+
+## Description
+
+This plugin provides a HMR (Hot Module Replacement) feature for @sapphire/framework. This allows you to add, delete, and
+update commands, listeners, and more without having to restart your bot. This allows for rapid iteration and prototyping
+without your bot slowing you down.
+
+## Features
+
+- Fully ready for TypeScript!
+- Includes ESM ready entrypoint
+- FAST!
+- Super easy to use
+
+## Installation
+
+`@sapphire/plugin-hmr` depends on the following packages. Be sure to install these along with this package!
+
+- [`@sapphire/framework`](https://www.npmjs.com/package/@sapphire/framework)
+
+You can use the following command to install this package, or replace `npm install` with your package manager of choice.
+
+```sh
+npm install @sapphire/plugin-hmr @sapphire/framework
+```
+
+---
+
+## Usage
+
+### JavaScript
+
+in your main or setup file, register the plugin:
+
+```javascript
+require('@sapphire/plugin-hmr/register');
+```
+
+if you want to make sure its only loaded in development, you can do this:
+
+```javascript
+const { start } = require('@sapphire/hmr');
+
+if (process.env.NODE_ENV === 'development') start();
```

Comment:
Clarify with a comment that this needs to be called after `SapphireClient#login`, so it registers the hooks for all the stores and all their paths (user paths are registered in login).

Perhaps something like:

```javascript
// Define client and everything

async function main() {
	await client.login();

	if (process.env.NODE_ENV === 'development') start();
}

void main();
```